tests/ci/common_posix_setup.sh

```diff
@@ -53,6 +53,21 @@ function build_and_test {
 
 function fips_build_and_test {
   run_build "$@" -DFIPS=1
+  # Upon completion of the build process. The module’s status can be verified by 'tool/bssl isfips'.
+  # https://csrc.nist.gov/CSRC/media/projects/cryptographic-module-validation-program/documents/security-policies/140sp3678.pdf
+  # FIPS mode is enabled when 'defined(BORINGSSL_FIPS) && !defined(OPENSSL_ASAN)'.
+  # https://github.com/awslabs/aws-lc/blob/220e266d4e415cf0101388b89a2bd855e0e4e203/crypto/fipsmodule/is_fips.c#L22
+  expect_fips_mode=1
+  for build_flag in "$@"
+  do
+    if [[ "${build_flag}" == '-DASAN=1' ]]; then
+      expect_fips_mode=0
+      break
+    fi
+  done
+  module_status=$(./test_build_dir/tool/bssl isfips)
```

Comment:
So now this is tested for both shared and static FIPS builds?

Reply:
Yes.